index.js

```diff
@@ -68,7 +69,44 @@ function toggleColor(element) {
   }
 }
 
+const h1Header = document.querySelector("#header")
+// h1Header.addEventListener("click", toggleColor)
```

Comment:
This won't work because toggleColor expects an element, which is different from an event

Reply:
Inside the callback function in your current solution, console.log the `event` and the `h1Header`. toggleColor expects and only knows how to work with `h1Header`, and doesn't have any logic to handle the`event` itself